handler/testdata/handler/bar/handler.go

```diff
@@ -0,0 +1,13 @@
+package bar
```

Comment:
I'd put it in `handler/internal/test/handler/bar/handler.go` to make it inaccessible outside.

Reply:
Moved around.